media-downloader/cmd/sync.go

```
@@ -0,0 +1,559 @@
+				client.SetProxy(proxies[index])
+			}
+			return client
+		},
+		ExecuteRequest: executeTmdbRequest,
+	})
+	defer rm.Close()
+
+	for i := range payloads {
+		payload := payloads[i]
+
+		rm.AddRequest(kyro.APIRequest[MediaPayload]{
```

golangci-lint (1.64.8)

Error return value of `rm.AddRequest` is not checked

(errcheck)